src/console.ml

```diff
@@ -0,0 +1,83 @@
+open! Stdune
+
+type status_line_config =
+  { message   : string option
+  ; show_jobs : bool
+  }
+
+module T = struct
+
+  type t = {
+    display : Config0.Display.t;
+    mutable status_line : string;
+    mutable gen_status_line : unit -> status_line_config;
+  }
+
+  let hide_status_line s =
+    let len = String.length s in
+    if len > 0 then Printf.eprintf "\r%*s\r" len ""
+
+  let show_status_line s =
+    prerr_string s
+
+  let update_status_line t ~pending_jobs =
```

Comment:
`pending_jobs` is actually the number of jobs currently running. It should probably be called `running_jobs`

Reply:
Renamed.